install_shiftit.sh

```diff
@@ -1,5 +0,0 @@
-curl -L https://raw.github.com/onsi/ShiftIt/master/ShiftIt.zip -o ShiftIt.zip
-unzip ShiftIt.zip ShiftIt.app/* -d /Applications/
-rm ShiftIt.zip
-
-open /Applications/ShiftIt.app/
```

Comment:
this is a fork that add features like multiple window sizes, that isn't available in the main branch (the one on homebrew), so i'd prefer to continue to use this